crates/aptos/src/node/local_testnet/ready_server.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::node::local_testnet::health_checker::HealthChecker;
+use anyhow::Result;
+use clap::Parser;
+use poem::{
+    get, handler,
+    http::StatusCode,
+    listener::TcpListener,
+    middleware::Tracing,
+    web::{Data, Json},
+    EndpointExt, IntoResponse, Route, Server,
+};
+use serde::Serialize;
+use std::net::{Ipv4Addr, SocketAddrV4};
+
+#[derive(Debug, Clone, Parser)]
+pub struct ReadyServerConfig {
+    #[clap(long, default_value_t = 8090)]
+    pub ready_server_listen_port: u16,
+}
```

Comment:
So, there's now yet another service just to check if the other services are running?

Could we shove this into an existing port?

So, now it looks like the local testnet node opens probably like 10 ports, between the node, the faucet, the grpc endpoint, the API, and the ready server.

Reply:
Why shove this into an existing service when it doesn't belong there? I don't think opening a bunch of ports is a big deal.